serde-proto/src/lib.rs

```diff
@@ -0,0 +1,8 @@
+mod timestamp;
+
+/// Event module for the autogenerated event code.
+pub mod common_types {
+    #![allow(clippy::derive_partial_eq_without_eq)]
+    #![allow(clippy::large_enum_variant)]
+    tonic::include_proto!("v1.common_types");
+}
```

Comment:
Control what we export?
```suggestion
}
pub use common_types::Timestamp;
```